docker/run-tests.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+docker-compose build joomla-base
+docker-compose up -d joomla-test
+sleep 20
+
+# Coding Standards.
+docker-compose exec joomla-test sh -c "phpcs ./yoti-joomla"
+
+# Run Tests.
+docker-compose exec joomla-test ./vendor/bin/codecept run acceptance --steps
```

Comment:
Same here RE pausing at the end

Reply:
Disabling pseudo-tty allocation in 24a6fb5 should resolve the Windows issues